public/index.ejs

```diff
@@ -1,23 +1,20 @@
 <!DOCTYPE html>
 <html lang="en-US">
+  <head>
+    <meta charset="utf-8" />
+    <title>Migration toolkit for Containers</title>
```

Comment:
Should be "Migration Toolkit" uppercase T

Reply:
updated